fail2ban/tests/clientreadertestcase.py

```diff
@@ -331,16 +334,34 @@ def testFilterReaderSubstitionDefault(self):
 
 	def testFilterReaderSubstitionSet(self):
 		output = [['set', 'jailname', 'addfailregex', 'to=sour@example.com fromip=<IP>']]
-		filterReader = FilterReader('substition', "jailname", {'honeypot': 'sour@example.com'})
-		filterReader.setBaseDir(TEST_FILES_DIR)
+		filterReader = FilterReader('substition', "jailname", {'honeypot': 'sour@example.com'},
+		  share_config=self.__share_cfg, basedir=TEST_FILES_DIR)
```

Comment:
again -- my comment is just about formatting.  If argument list expands to more than 1 line, best would be to start listing all arguments from the new line so it is easier to read the code, imho.